src/urg_stamped.cpp

```diff
@@ -837,10 +882,9 @@ UrgStampedNode::UrgStampedNode()
                   boost::arg<2>(),
                   boost::arg<3>()));
 
-  if (clock_estim_interval > 0.0)
+  if (clock_estim_interval_ > ros::Duration(0))
   {
-    timer_delay_estim_ = nh_.createTimer(
-        ros::Duration(clock_estim_interval), &UrgStampedNode::estimateSensorClock, this);
+    next_sync_ = ros::Time::now() + clock_estim_interval_;
```

Comment:
Does this mean that `next_sync_` might be the same time for multiple sensors on start up?

Reply:
Yes. Timing will be changed after a collision is detected